__tests__/marker/table-component.ts

```diff
@@ -0,0 +1,26 @@
+import { marker } from '@ngneat/transloco-keys-manager/marker';
```

Comment:
replace the import with `@ngneat/transloco-keys-manager`

Reply:
no, i use this fork in production, for frontend and backend, backend webback work correct, frontend try serve webpack import

Reply:
and fail

Reply:
I don't understand what's the issue here, there is already a test case for importing this function from `@ngneat/transloco-keys-manager/marker` in `table-component-with-alias-t` I just want the other import case to be tested as well.

Reply:
I'm not sure I understand how this is related since this is a test file. I just want the import from `@ngneat/transloco-keys-manager` to be tested as well.

Reply:
for correct work marker need import is as standalone for use without webpack sources

Reply:
some users use test cases as manual for use, ```import { marker } from '@ngneat/transloco-keys-manager';``` - it is incorrect use of marker

Reply:
@shaharkazaz

Reply:
Leave it as is but remove the export from the public API file.

Reply:
done